EasyPost/Parameters/V2/Addresses.cs

```diff
@@ -0,0 +1,79 @@
+using EasyPost.Utilities.Annotations;
```

Comment:
I thought we were going to break this up into smaller PRs? It's difficult to review individual properties of so many resources in a single go. I'd suggest keeping `Address` here to prove the concept with the guts/internals of the logic, then breaking the rest of the resources into two separate PRs that build on top of this one once merged.

Reply:
Fair enough, was a lot of copy-pasta. Will remove all classes except Address Create and Shipment Create, used in the current unit tests.